docker-compose.yml

```
@@ -9,15 +9,17 @@ services:
     volumes:
       - "beacon-chain-data:/data"
     ports:
-      - "12000/udp"
-      - "13000/tcp"
+      - "12004:12004/udp"
```

```suggestion
      - "13003:13003/udp"
```
this is the specified port.